azavea/district-builder-dtl-pa
# Fix hostnames within container

## Overview

Make Django reflect the host that is accessed outside of the container.

### Checklist


## Testing Instructions

* On `develop` branch, start server
* Make this change:
```
diff --git a/django/publicmapping/publicmapping/templates/index.html b/django/publicmapping/publicmapping/templates/index.html
index cde93d3..aff2079 100644
--- a/django/publicmapping/publicmapping/templates/index.html
+++ b/django/publicmapping/publicmapping/templates/index.html
@@ -79,7 +79,7 @@
       <div id="logo">
                
               <span style="font-size:250%; font-family:arial,sans-serif; font-weight:bold; color:white">
-                 {% trans "DistrictBuilder" %}
+                 {{ request.build_absolute_uri }}
                </span>
       </div>
       <div id="site_nav">
```
* `docker-compose restart django`
* Reload homepage and notice that the page now shows `http://web:8080/` as the title
* Check out this branch
* Stop server, run `docker-compose build` to rebuild nginx config, and restart server
* Reload homepage and notice that the page now shows `http://localhost:8080/` as the title
